src/frontend/apps/desk/src/features/teams/components/Panel/TeamItem.tsx

```diff
@@ -88,6 +88,9 @@ export const TeamItem = ({ team }: TeamProps) => {
           <Text
             $weight="bold"
             $color={!hasMembers ? colorsTokens()['greyscale-600'] : undefined}
+            $css={`
+              text-wrap: nowrap;
```

Comment:
The prob with `nowrap` when the team name will be a bit long it will be out of the container.
![image](https://github.com/numerique-gouv/people/assets/25994652/5081c111-acbe-45bf-b607-f4e62cce7263)